portfolio/src/main/java/com/google/sps/data/SchedulingCompleteness.java

```diff
@@ -0,0 +1,7 @@
+package com.google.sps.data;
+
+public enum SchedulingCompleteness {
```

Comment:
You can consider assigning integer values, so that you don't have to perform string comparisons, and client can work with the int values.

ex:
public enum SchedulingCompleteness {
  NOT_SCHEDULED(0),
  ...(1),
  ...(2);

  private final int value;

  SchedulingCompleteness(int value) {
    this.value = value;
  }

  public int getValue() {
    return this.value;
  }
}

Reply:
I updated the enum accordingly. Right now, ScheduledTask takes in `Optional<Integer>` as the representation for scheduling completeness -- would this make sense? 
Another question I had was, based on the current implementation, I don't think the Optional object is necessary anymore? This completeness field will always have a value.

Reply:
I see. Yes I agree the optional is no longer necessary. Please see my comment below on using enum and integer.